### ISSUE TITLE
req.conversation.postActivityToUser is not a function

### ISSUE THREAD

**Issue by TemoeBOUISSOU** [microsoft/BotFramework-Emulator#2109]
## Version
4.8.0

## Describe the bug
I want to save then resume a conversation by applying **SendConversationHistoryAsync** method with a new **Transcript** object (not saved by the middleware) filled with a custom list of activities and the right conversation ID. When the emulator receives that request, i have an error which was not here in 4.7.
"Processed 0 of 4 activities successfully.TypeError: req.conversation.postActivityToUser is not a function"

## To Reproduce
1. Save conversation ID with the list of activities in storage
2. Retrieve it from the storage
3. Post it back to the user/emulator by using SendConversationHistoryAsync
4. "Processed 0 of 4 activities successfully.TypeError: req.conversation.postActivityToUser is not a function" error on the emulator

## Expected behavior
I want the emulator to replay the exact list of activities i post to the emulator using the SendConversationHistoryAsync method. **I know emulator is for testing purpose**. If i am wrong in the implementation of that behavior, please explain me why !

## Screenshots
![Bug emulator](https://user-images.githubusercontent.com/37652994/76803626-5f432e80-677e-11ea-8bdd-7739da748da3.jpg)

## Additional context
1. It is sent using the **Conversations** property of a newly **ConnectorClient** class instantiated with the right values.
2. I tried to change the information inside activities like updating activities id, timestamps. But i did try other properties without success (properties which i thought could be responsible).
3. In the 4.7, i had the activities showing but not user sent activities, only bot activities were displayed.
4. I did not tried to debug emulator code on that scenario.
5. I am kind of new using all of these tools, excuse me if i went wrong !

[bug]

**Comment by dmvtech** [comment]
Hi @TemoeBOUISSOU 

I'm trying to understand the use case. Maybe understanding that will help figure out if this is the best route or not.

To be clear; you're not just wanting to inspect conversations using the emulator. Correct?

There is inspection middleware. Which _might_ help in doing what you need. 
https://docs.microsoft.com/en-us/azure/bot-service/bot-service-debug-inspection-middleware?view=azure-bot-service-4.0&tabs=csharp

But again, it depends on what you are really wanting to do.

**Comment by cwhitten** [comment]
Hello @TemoeBOUISSOU, we [recently introduced the ability](https://github.com/microsoft/BotFramework-Emulator/pull/2089) to replay all activities up to a certain point in a conversation when restarting from the Emulator. Would this cover your scenario?

<img width="665" alt="Screen Shot 2020-03-17 at 1 32 38 PM" src="https://user-images.githubusercontent.com/1156704/76898912-d4346800-6853-11ea-906c-e4e39617aaa4.png">

**Comment by TemoeBOUISSOU** [comment]
Thank you for your answers ! Just to explain a little bit more, i want to build my own structure to store the conversations data (in database) in order to replay it to the user on the fly. The objective might be explained more with that schema.

![ConvTree](https://user-images.githubusercontent.com/37652994/76902534-4a2ed400-6840-11ea-8f8f-f5d7e989a7e2.jpg)

Since the bug report, i took the time to inspect emulator code between **4.7** and **4.8** on this subject and i can tell that my error is coming from the [Conversation.ts](https://github.com/microsoft/BotFramework-Emulator/blob/v4.7.0/packages/app/main/src/server/state/conversation.ts) file as the [sendHistoryToConversation.ts](https://github.com/microsoft/BotFramework-Emulator/blob/v4.7.0/packages/app/main/src/server/routes/channel/conversations/handlers/sendHistoryToConversation.ts) file is using it.
I succeeded to change the emulator code to do exactly what i need but i think that it's not a wanted behavior as the method **SendConversationHistoryAsync** in c# framework is designed for this.

To be precise, i don't want to use the emulator for long-term usage of this, but i use emulator for testing purposes, validating some of possibilities that bot framework is bringing. I might think that this feature is something that was already implemented but has been deleted since **4.8**. It is just a way to send a full conversation history to the emulator and replay it according to the conversation id and a list of activities.

I succeeded to implement all of this since my bug report but i had to change the emulator code and it is not what i plan to work with the emulator. I don't want to maintain that behavior myself as i think that it's something you wanted to keep (not sure) ! So for now i have my problem solved but the bug remain in the emulator code.
